### PR TITLE
Run Prettier on the rest of the things

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

File: doc/TUTORIAL.md [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
 
 You should read [CONTRIBUTING.md](../CONTRIBUTING.md)
 
 You can also read previous
 [merged pull-requests with the 'service-badge' label](https://github.com/badges/shields/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Aservice-badge+is%3Amerged)
 to see how other people implemented their badges.
 
-(2) Setup
----------
+## (2) Setup
```
Comment (calebcartwright) [inline doc/TUTORIAL.md]: What would folks think about adding the url for the Github workflow to https://guides.github.com/activities/forking/ or changing the existing url? It is currently http://try.github.io/ which is a great resource for git

File: doc/TUTORIAL.md [hunk 2]
```diff
@@ -96,32 +93,35 @@ Each service has a directory for its files:
...
   class for the badges or one may already exist.
 
-[BaseService]: https://github.com/badges/shields/blob/master/services/base.js
+[baseservice]: https://github.com/badges/shields/blob/master/services/base.js
 
 As a first step we will look at the code for an example which generates a badge without contacting an API.
 
 ```js
-'use strict'                                         // (1)
+'use strict' // (1)
```
[the remark this answers is not in the document]
Comment (paulmelnikow) [reply doc/TUTORIAL.md]: Hrm, that is not so great. I'm remembering now why I didn't turn this on before. I'll reset and exclude this file.

File: doc/service-tests.md [hunk 3]
```diff
@@ -53,28 +50,30 @@ First we'll add a test for the typical case:
 ```js
 const { isBuildStatus } = require('../test-validators')
 
-t.create('Build status')                     // 1
+t.create('Build status') // 1
   .get('/build/wercker/go-wercker-api.json') // 2
-  .expectJSONTypes(                          // 3
-    Joi.object().keys({                      // 4
-      name: 'build',                         // 5
```
[the remark this answers is not in the document]
Comment (paulmelnikow) [reply doc/service-tests.md]: I feel like we could remove the numbered lines in this one. They make it extra clear what corresponds to what, but I think the explanation could be just as easy to follow without them.

File: gh-badges/lib/colorscheme.json [hunk 4]
```diff
@@ -1,13 +1,13 @@
 {
-  "brightgreen":    { "colorB": "#4c1" },
```
[the remark this answers is not in the document]
Comment (paulmelnikow) [reply gh-badges/lib/colorscheme.json]: It's reasonable to argue that the old version was prettier. Though this is the point of autoformatting. Everyone has their own opinion. Everything that gets auto-formatted looks fairly decent, and doesn't require any work to debate or maintain.

[the remark this answers is not in the document]
Comment (calebcartwright) [reply gh-badges/lib/colorscheme.json]: Works for me! Just wanted to check

File: doc/service-tests.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (calebcartwright) [reply doc/service-tests.md]: Yeah I think that will work here

File: doc/TUTORIAL.md (hunk 1 above)
Comment (calebcartwright) [reply doc/TUTORIAL.md]: K I'll plug these in shortly

Comment (calebcartwright) [reply doc/TUTORIAL.md]: Gotcha, yeah I'll do that in a follow up (at some point)

**Comment by chris48s** [comment]
I had no idea running prettier on markdown files was a thing. Every day's a school day :school:

**Comment by paulmelnikow** [comment]
It's really handy! especially for dealing with code examples that are embedded in Markdown.

**Pull request merged**